components/03-modules/modal/modal--header-footer.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if trigger }}
+    {{ render '@modal-trigger' modalTrigger }}
+{{/if}}
+<div role="dialog"
+     aria-hidden="true"
+     id="{{ modal.id }}"
+     data-modal="{{ modal.id }}"
+     class="modal {{ modal.class }}"
+     tabindex="-1"
+     {{{ modal.attributes }}}
+ >
+    <div role="document"
+         class="modal__content {{ modalContent.class }}"
+         {{{ modalContent.attribtues }}}
+         tabindex="0"
+     >
+        <div class="modal__header">
+            {{{ header.html }}}
+            {{#if header.content }}
+                {{ render (component header.content) header.contentContext }}
+            {{/if}}
+            {{#if buttonClose }}
+                {{ render '@button--icon' buttonClose }}
+            {{/if}}
+        </div>
+        <div class="modal__main">
+            {{{ main.html }}}
```

Comment:
I'd add the `if` statement here

Reply:
but if main.html = '' it won't display so why put iff ?

Reply:
you're right, but I think we should change it to some "description" or smth like that, like we were speaking on the meeting